purescript.cabal

```diff
@@ -2,7 +2,7 @@ cabal-version:  2.4
 
 name:           purescript
 -- note: When updating the prerelease identifier, update it in app/Version.hs too!
-version:        0.14.5
+version:        0.15.0
```

Comment:
We changed this because it made testing much easier so we could see that the right compiler and package sets were used. Is there any advantage in removing it now and adding it back in a subsequent PR/commit?

Reply:
Let me rephrase: will the build break if this is not merged with the PR? We have a whole release process during which version numbers are updated; it's not like you're responsible for bumping the version number just because you're making a breaking change. But if there's something about the parallel package set you're using that requires this for CI to pass, then we should know about that.

Reply:
The parallel package sets  [`metadata.version` to `v0.15`](https://github.com/working-group-purescript-es/package-sets/blob/main/packages.dhall#L111). But if I'm not mistaken, we are not using spago package-sets here in the compiler repo, but rather bower. So I think it will not break, the only thing is that the master branch will not work with the package-set from the working group until the release increases the version.

Reply:
Will remove this then.

Reply:
Done